packages/odyssey-react/src/components/RadioInput/index.tsx

```diff
@@ -0,0 +1,143 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React, { useCallback } from 'react';
+import type {
+  FunctionComponent,
+  FocusEventHandler,
+  ChangeEvent,
+  RefCallback,
+} from 'react';
+import { useOid } from '../../utils';
+
+export type Props = {
+  /**
+   * The underlying input element id attribute. Automatically generated if not provided
+   */
+  id?: string,
+
+  /**
+   * The form field label
+   */
+  label: string,
+
+  /**
+   * Callback to provide a reference to the underlying input element
+   * @param {Object} instance the input element or null
+   */
+  inputRef?: RefCallback<HTMLInputElement>,
+
+
+  /**
+   * The underlying input element name attribute
+   */
+  name: string,
+
+  /**
+   * The underlying input element required attribute
+   * @default true
+   */
+  required?: boolean,
+
+  /**
+   * The underlying input element disabled attribute
+   * @default false
+   */
+  disabled?: boolean,
+
+  /**
+   * The underlying input element value attribute
+   */
+  value: string,
+
+  /**
+   * The input element checked state for controlled components
+   * @default false
+   */
+  checked?: boolean,
+
+  /**
+   * Callback executed when the input fires a blur event
+   * @param {Object} event the event object
+   */
+  onBlur?: FocusEventHandler<HTMLInputElement>,
+
+  /**
+   * Callback executed when the input fires a change event
+   * @param {Object} event the event object
+   * @param {string} value the string value of the input
+   */
+  onChange?: (event: ChangeEvent<HTMLInputElement>, value: string) => void,
+
+  /**
+  * Callback executed when the input fires a focus event
+  * @param {Object} event the event object
+  */
+  onFocus?: FocusEventHandler<HTMLInputElement>,
+};
+
+
+/**
+ * Text inputs allow users to edit and input data.
+ */
```

Comment:
```suggestion

/**
 * Radios appear as a ring-shaped UI accompanied by a caption that allows the user to choose only one option at a time.
 *
 * @component
 * @example
 * <RadioInput label="Input Label" name="input_name" value="input_value" />
 */
```

Reply:
This text comes from the lede in our [docs site](https://odyssey.okta.design/components/radio-button/)

Reply:
nice I forgot to update this good catch. What is the requirement for the additional JSDoc annotations?

They aren't parsed by the storybook generator from what I've seen - are you suggesting this to support developers using editor plugins?

Reply:
That is the practical reason to do it today, yes. Longer-term, I would love for us to explore parsing these comments to compose our docs site pages!